apps/condo/domains/property/schema/Property.js

```diff
@@ -288,6 +289,14 @@ const Property = new GQLListSchema('Property', {
                 scale: 2,
             },
         },
+
+        mapHash: {
+            schemaDoc: 'Hash sum of property map field',
+            type: Virtual,
+            resolver: (property) => {
+                return crypto.createHash('md5').update(JSON.stringify(property.map)).digest('hex')
```

Comment:
wouldn't it be better to calculate this on change and store then ?

Reply:
we relly don't need to search